azext_iot/central/_help.py

```diff
@@ -44,6 +57,288 @@ def load_central_help():
+        short-summary: Get an export details
+        examples:
+        - name: Get an export details
+          text: >
+            az iot central export show
+            --app-id {appid}
+            --export-id {exportid}
+    """
+
+    helps[
+        "iot central export create"
+    ] = """
+        type: command
+        short-summary: Create an export for an IoT Central application.
+        examples:
+        - name: Create an export from file
+          text: >
+            az iot central export create
+            --app-id {appid}
+            --export-id {exportid}
+            --content './filepath/payload.json'
+
+        - name: Create an export from json payload
+          text: >
+            az iot central export create
+            --app-id {appid}
+            --export-id {exportid}
+            --content "{
+              'displayName': 'Test Export 1',
+              'enabled': false,
+              'source': 'telemetry',
+              'filter': 'SELECT * FROM devices WHERE $displayName != \'abc\' AND $id = \'a\'',
+              'enrichments': {
+                'abc': {
+                  'path': '$templateDisplayName'
+                }
+              },
+              'destinations': [
+                {
+                  'id': 'fa5792a4-ead3-41dc-a972-fbeed33d46ae',
+                  'transform': '{ ApplicationId: .applicationId, Component: .component, DeviceName: .device.name }'
+                }
+              ]
+            }"
+    """
+
+    helps[
+        "iot central export update"
+    ] = """
+        type: command
+        short-summary: Update an export for an IoT Central application.
+        examples:
+        - name: Update an export from file
+          text: >
+            az iot central export update
+            --app-id {appid}
+            --export-id {exportid}
+            --content './filepath/payload.json'
+
+        - name: Update an export's display name and enable export from json payload
+          text: >
+            az iot central export update
+            --app-id {appid}
+            --export-id {exportid}
+            --content "{'displayName': 'Updated Export Name', 'enabled': true}"
+    """
+
+    helps[
+        "iot central export delete"
+    ] = """
+        type: command
+        short-summary: Delete an export for an IoT Central application.
+        examples:
+        - name: Delete an export
+          text: >
+            az iot central export delete
+            --app-id {appid}
+            --export-id {exportid}
+    """
+    _load_central_destination_help()
+
+
+def _load_central_destination_help():
+    helps[
+        "iot central export destination"
+    ] = """
+        type: group
+        short-summary: Manage and configure IoT Central export destinations.
+    """
+
+    helps[
+        "iot central export destination list"
+    ] = """
+        type: command
+        short-summary: Get the list of export destinations for an IoT Central application.
```

Comment:
If list does not take in an optional query, make sure to say get the list of _all_ export destinations to let the user know.

Reply:
Fixed.